### DISCUSSION TITLE
使用SetupAttachment调整组件层级

### DISCUSSION THREAD

**Discussion by xiaochen-CN** [Tencent/puerts#1659] [Q&A]
![image](https://github.com/Tencent/puerts/assets/62801004/a9366ef2-87e5-4f63-a606-58d9cdf064b2)

![image](https://github.com/Tencent/puerts/assets/62801004/024eca19-0a43-4f60-9b96-88c847cf4d69)

似乎SetupAttachment没法用来调整组件层级？
或是我使用错误？

**Comment by chexiongsheng** [comment]
puerts并未实现任何ue引擎的功能。主要提供了个ts/js环境以及脚本和引擎间的互相调用支持。
某个api该哪里调，怎么调，调用后有什么效果请查UE的api文档。
